cluster/azure/util.sh

```diff
@@ -493,7 +493,7 @@ function kube-down {
 #  echo
 #  echo "  https://${KUBE_MASTER_IP}"
 # echo
-#  echo "The user name and password to use is located in ~/.kubernetes_auth."
+#  echo "The user name and password to use is located in $DEFAULT_KUBECONFIG."
```

Comment:
I think we should write here ${KUBECONFIG}, not ${DEFAULT_KUBECONFIG}. If someone has redefined KUBECONFIG variable before starting the cluster, we should write it, not the default value.